docs/source/_extensions/accordion.py

```diff
@@ -0,0 +1,69 @@
+"""Add an ``accordian`` directive.
```

Comment:
Is this a typo (i.e., should be accordi**o**n), or on purpose? Same for the filenames `accordian.js` and `accordian.css`.

Reply:
yes, I seem to have a specific dyslexia when it comes to spelling accordion lol!

Reply:
fixed in 80449ce81809c41d11ebf223a1311f9c7fa42e6e